_i18n/en.yml

```diff
@@ -107,9 +107,30 @@ faq:
     question: IRMA is <em>open source</em>; what does that mean?
     answer: The IRMA software is deliberately open source. This means that the way that IRMA works can be inspected by anyone<span>:</span> the software is available online, see <a href="https://github.com/privacybydesign" target="_blank">GitHub</a>. This also means that people outside the foundation can contribute. Indeed, several other organisations are contributing software and designs to the IRMA ecosystem. Of course, you do have to know a bit about computer programs in order to really understand IRMA's software, but the principle is important<span>:</span> IRMA has no secrets and works in a transparent manner. This contributes to trust in IRMA. There are no uneasy discussions about hidden backdoors in IRMA, like for instance in 5G telecommunications. The Privacy by Design foundation is of the opinion that all software in infrastructure with a public role should be open source.
   item17:
+    question: I would like to remove IRMA and all my data.
+    answer: At <a href="https://privacybydesign.foundation/myirma/">MijnIRMA</a> you can remove your data. Then, delete the IRMA app from your phone. If you don't use IRMA for a year, we automatically delete all your data.
+  item18:
+    question: I want to unsubscribe at an organization I log in at with IRMA.
+    answer: Please contact the organization you wish to unsubscribe from.
+  item19:
+    question: I want to log in or view my data at an organization that uses IRMA. How do I do that?
+    answer: Follow the instructions on the organization's website. If you have any questions about this, please contact the organisation.
+  item20:
+    question: How do I scan the IRMA QR code?
+    answer: Open the IRMA app with your PIN code and press the &lsquo;Scan QR&rsquo; button at the bottom right of the app. Scan the QR code by pointing your mobile at the IRMA QR code. Sometimes it is necessary to give permission to use the camera.
+  item21:
+    question: Where can I find an IRMA QR code on an organization's website?
+    answer: You can usually find the IRMA QR code via a button or link &lsquo;Login&rsquo; or &lsquo;Share data&rsquo;. Sometimes the organization does not support IRMA (yet). If you have any questions about this, please contact the organisation.
+  item22:
+    question: What does the &lsquo;Assurance level&rsquo; in the &lsquo;Personal data&rsquo; card mean?
```

Comment:
In English eIDAS terms `level of assurance` is used. However, I see that this term is also used in the scheme manager. Maybe adjust some time in the future?